packages/collapsible/rc/index.js

```diff
@@ -50,7 +50,7 @@ const waitForHeightTransitionToEnd = element => {
 }
 const updateOverflowStyle = ({ element, open, isTransitioning = false }) => {
   if (element) {
-    element.style.overflow = isTransitioning || !open ? 'hidden' : 'visible'
+    element.style.overflow = isTransitioning || !open ? 'hidden' : 'auto'
```

Comment:
If a user places a paragraph or another child element with margins inside a collapsible parent this style will prevent unexpected rendering such as gaps from appearing such as in the Drawer.